test/unit/auth/user-record.spec.ts

```diff
@@ -653,6 +653,15 @@ describe('UserRecord', () => {
     });
   });
 
+  describe('clears REDACTED passwordHash', () => {
```

Comment:
This seems like a test of `passwordHash` getter.
Can you move it next to `it('should return expected undefined passwordHash', () => {`
You can define something like: `it('should clear REDACTED passwordHash', () => {`

Reply:
Done